source/components/MetaLink.svelte

```diff
@@ -0,0 +1,9 @@
+<h3 class="sr-only">Dependency List via N P M Hub</h3>
+<div class="mt-3">
+  <a href="#npmhub" class="Link--muted">
+    <svg version="1.1" aria-hidden="true" height="16" width="16" viewBox="150 50 250 450" data-view-component="true" class="octicon mr-2">
```

Comment:
What's that data view component attribute?

Reply:
I duplicated the SVG attributes from the other octicons. I was thinking that GitHub might have some use for it like the `data-pjax-*` attribute. If it is irrelevant, I can remove it.

Reply:
You can get the actual icons from https://primer.style/octicons/

Reply:
octicons would not have the icon for npmhub :confused: 

What I meant was that I _copied_ the SVG **attributes** that were on the octicons ( for Readme and License ) and used them on the npmhub icon.

Did you not notice `Promise { <resolved> }` on the screenshot above? :stuck_out_tongue_closed_eyes:

PS : I know about primer.style. I have it open usually when I am contributing to Refined GitHub.

Reply:
> Did you not notice `Promise { <resolved> }` on the screenshot above? 😝

well don't do that, no idea of what it means. Just send a comment when you have the screenshot

Reply:
But I see it now. I like the icon 👍